internal/pkg/addon/storage.go

```diff
@@ -147,16 +159,28 @@ func NewDDBTemplate(input *DynamoDBProps) *DynamoDBTemplate {
 
 // RDSProps holds RDS-specific properties for addon.NewRDSTemplate().
 type RDSProps struct {
-	WorkloadType   string   // The type of the workload associated with the RDS addon.
-	ClusterName    string   // The name of the cluster.
-	Engine         string   // The engine type of the RDS Aurora Serverless cluster.
-	InitialDBName  string   // The name of the initial database created inside the cluster.
-	ParameterGroup string   // The parameter group to use for the cluster.
-	Envs           []string // The copilot environments found inside the current app.
+	WorkloadType            string   // The type of the workload associated with the RDS addon.
+	ClusterName             string   // The name of the cluster.
+	AuroraServerlessVersion string   // The version of Aurora Serverless
```

Comment:
nit: can be private, and can we add [a period](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences) at the end of the comment plz 🥺 
```suggestion
	auroraServerlessVersion string   // The version of Aurora Serverless.
```

Reply:
changed into private field and added a period.